<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserCenterWidget</class>
 <widget class="QWidget" name="UserCenterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1002</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1002</width>
    <height>6200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EntCenterWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EntCenterWidget{background-color: rgb(255, 255, 255);};</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="EntLeftWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>618</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>618</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#EntLeftWidget{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #EDF4FF, stop:1 #FAFCFF);border-top-left-radius:8px;border-bottom-left-radius:8px;border-right:1px solid #EBEBEB;}</string>
       </property>
       <widget class="AvatarLabel" name="labelAvatar">
        <property name="geometry">
         <rect>
          <x>95</x>
          <y>30</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/EntCenterWidget/image/img_tx_d.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelUserName">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>100</y>
          <width>210</width>
          <height>35</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px &quot;微软雅黑&quot;;
font-weight:bold;
color:#333333;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labelUserID">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>144</y>
          <width>150</width>
          <height>12</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>12</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:#333333;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelSpaceLeft">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>221</y>
          <width>250</width>
          <height>1</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(222, 222, 222);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="scoreButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>251</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip">
         <string>云币</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/score.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="couponButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>251</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip">
         <string>优惠劵</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/coupon.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelMore">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>332</y>
          <width>57</width>
          <height>15</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>57</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px &quot;微软雅黑&quot;;
font-weight:bold;
color:#333333;</string>
        </property>
        <property name="text">
         <string>更多信息</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelEntInfo">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>371</y>
          <width>10</width>
          <height>10</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgba(223,70,100,1);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="mainButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>360</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>570</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(51, 51, 51, 1);padding-left:-20px;}
QPushButton:hover,pressed{ background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(31,139,237,1);}</string>
        </property>
        <property name="text">
         <string>首        页</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S, Esc</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelScore">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>281</y>
          <width>50</width>
          <height>13</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>13</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16px &quot;微软雅黑&quot;;
color:#666666;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelCoupon">
        <property name="geometry">
         <rect>
          <x>155</x>
          <y>280</y>
          <width>50</width>
          <height>13</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>13</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16px &quot;微软雅黑&quot;;
color:#666666;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelUser">
        <property name="geometry">
         <rect>
          <x>95</x>
          <y>30</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/EntCenterWidget/image/qyzx_icon_vip_wjr.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonLogin">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>130</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#1E8BED;</string>
        </property>
        <property name="text">
         <string>立即登录</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>210</width>
          <height>35</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px &quot;微软雅黑&quot;;
color:#333333;</string>
        </property>
        <property name="text">
         <string>登录后可体验更多功能</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labelEntInfo_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>422</y>
          <width>10</width>
          <height>10</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgba(223,70,100,1);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAcc">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>410</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>570</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(51, 51, 51, 1);padding-left:-20px;}
QPushButton:hover,pressed{ background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(31,139,237,1);}</string>
        </property>
        <property name="text">
         <string>账号设置</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S, Esc</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="RightWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>750</width>
         <height>618</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>750</width>
         <height>618</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#EntRightWidget{background-color: rgb(252, 252, 252);border-top-right-radius:8px;border-bottom-right-radius:8px;}</string>
       </property>
       <layout class="QHBoxLayout" name="rightLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="layout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AvatarLabel</class>
   <extends>QLabel</extends>
   <header location="global">AvatarLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
